.k8s/components/cache/nginx.conf

```
@@ -0,0 +1,37 @@
+
+events {
+ worker_connections 1024;
```

Should be 2 spaces indentation no ?

```suggestion
  worker_connections 1024;
```